src/vellum/workflows/nodes/displayable/final_output_node/node.py

```diff
@@ -38,6 +42,60 @@ def get_output_type(cls) -> Type:
         else:
             return all_args[1]
 
+    @classmethod
+    def _validate_output_type_consistency(mcs, cls: Type) -> None:
```

Comment:
[nit] do we still plan on each node defining a `__validate__()` method or something like that? the final output node's validate method could then call this guy